src/platform/user/profile/vap-svc/components/ContactInformationFieldInfo/ContactInformationUpdateSuccessAlert.jsx

```diff
@@ -29,6 +29,7 @@ const ContactInformationUpdateSuccessAlert = ({ fieldName }) => {
         status="success"
         visible="true"
         full-width
+        slim
         uswds
```

Comment:
I think we can remove `uswds` prop since this is now the default for `va-alert`

Reply:
I'd hate to make this small change, and require everyone need to re-review the PR - we can update this in a future PR